### DISCUSSION TITLE
First Sketches

### DISCUSSION THREAD

**Discussion by IGIUK** [jelster/space-truckers#34] [Ideas]
![WhatsApp Image 2021-02-03 at 21 52 05 (2)](https://user-images.githubusercontent.com/75528351/106809483-c9a5db80-667c-11eb-8c2c-5da0a43a9f30.jpeg)
![WhatsApp Image 2021-02-03 at 21 52 05 (1)](https://user-images.githubusercontent.com/75528351/106809496-cdd1f900-667c-11eb-96d5-c45892e3d1a4.jpeg)
![WhatsApp Image 2021-02-03 at 21 52 05](https://user-images.githubusercontent.com/75528351/106809513-d32f4380-667c-11eb-9ec7-bbd948ac0ce2.jpeg)
![WhatsApp Image 2021-02-03 at 21 52 04 (1)](https://user-images.githubusercontent.com/75528351/106809519-d4607080-667c-11eb-9842-a3ad6e391607.jpeg)
![WhatsApp Image 2021-02-03 at 21 52 04](https://user-images.githubusercontent.com/75528351/106809523-d5919d80-667c-11eb-9741-b8b43c138659.jpeg)

**Comment by jelster** [comment]
I am amazed and impressed with these sketches - thanks for doing this! :rocket: :truck:

Here's my impressions and feedback on each:

## Meteors

Something I like a lot about these is how the two types (craters vs. bumps) help get the thematic elements across while being interestingly different from each other. The largely spherical shape suggests a certain size of asteroid; smaller ones are much more irregular in shape.

I'm curious what some of these smaller asteroids might look like, intermingled with these larger ones - this is a great direction!

## Alien landscape

This sketch has a lot of great things going on in it. The vague shapes in the background lend an air of mystery that complements the brightly-colored fauna (plant life, trees, etc) nicely. I can easily imagine an alien painter setting their easel up on this placid-looking nature path :smile:

As-is this is a great conceptual piece. I've been thinking on how we (as the player) might encounter this type of environment, here's a description of something to experiment with on this:

Let's imagine this scene depicts a receiving point for space-cargo. The actual landing site is over the hills to the distance, but from there goods are distributed along paths wide and narrow, like this one. A 👾 child waits on one of the stumps to the side of the well-worn path, stubby legs swinging off the side. They looks up, excitement brightening their eyes as sunlight catches off the space-windshield of the space-rig, newly arrived with a load of stuff. It's Space-Mail Day!

## Meteors

### Junk Rain

Yes! :heart:

### Carrot meteor

I don't think I would have ever have thought to put the words "carrot" and "meteor" together like this, but somehow it kinda works and it pretty exactly describes what it is! :smile:

This might need to be developed a bit further. There's something here with the idea of a meteor made of out stuff like carrots, but I'm not sure what it might be yet. 

Just spitballing, I feel like maybe some of the people in this universe are rabbits, right? Imagine a rabbit astronomer discovering one of those carrot meteors, right?

## Pilot

Awwww how cute! 😸 

There's a lot of great potential in these. I can see a construction crew all in their space-worksuits. I think that we might need to have a couple of different archetypes, with different body shapes, color and fur patterns between them. Something that I noticed in a slightly jarring way was that the tail isn't inside a space-suit. I think it might actually be a bit funny-in-a-cute-way to have the tail kind of wrapped in an extension of the suit.

## Space-Amoebas

I like the callout for making them a hazardous red tone - with emissive textures I think that could look really cool! Do you picture the amoebas' bodies as being transparent, or opaque? I think making them at least partially transparent (~0.65-0.78 or so) would be a nice way to give it an almost ethereal quality in certain angles and lighting (some great PBR fun potential here!)

**Comment by IGIUK** [comment]
![6c0fe261-e39b-4a2b-b55b-f9a577300fa7](https://user-images.githubusercontent.com/75528351/107391805-220c2b80-6b0a-11eb-92a2-c0999946a904.jpg)
![8bfb9ee3-7293-4825-9a89-97e5ea7ff5be](https://user-images.githubusercontent.com/75528351/107391820-25071c00-6b0a-11eb-90c4-570ca4a37ad3.jpg)
![79dd9902-68bb-4d78-9137-039f0d77da28](https://user-images.githubusercontent.com/75528351/107391826-259fb280-6b0a-11eb-8fdb-583bf0b66a9e.jpg)
![97fd1554-f3ba-4213-a7e9-7f0eec053bb9](https://user-images.githubusercontent.com/75528351/107391829-26384900-6b0a-11eb-8c6c-84b44c17d244.jpg)
![ca15b947-ca15-4f9e-8c73-ea9d7eb78323](https://user-images.githubusercontent.com/75528351/107391831-26d0df80-6b0a-11eb-930a-18eccbe9ea2d.jpg)
![ec4cc6b5-f8a4-4412-bddd-858f5ee37ca2](https://user-images.githubusercontent.com/75528351/107391832-26d0df80-6b0a-11eb-8f53-fe379b54add4.jpg)

**Reply by jelster** [reply]
Oh my - lots of great stuff to comment on here!

### Post-Stations (small and large)

I like how these convey the impression of a functional and busy structure - the cargo handling manipulators are an especially nice touch. The landing pad is nice when taken by itself, however I think it needs to be secondary to the (not currently depicted) mass-driver (catapult) launcher - the "G.O.O.S.E" system. Normally, I would be open to changing the launch concept but it forms a integral part of the game design (think phase 1 route planning). [This sketch](https://github.com/jelster/space-truckers/blob/main/design/mass-driver-concept.png) shows the launch ramp in foreshortened view - let's see how that might fit with the design?

### Carrot meteor and junk

Good evolution on this. I was thinking, maybe we could have the larger moons made out of cheese?

side note -- I liked the random rivets, screws, bolts, etc. that the prior iteration had, don't want to forget those!

### Spaceships

The bird-inspired design is really compelling and adds a lot of depth to the world. I picture these as being the "last mile (or km)" delivery services, that bring cargo delivered via space-truck to a central station to the far-flung reaches of the planet or moon. Fleets of these could flock around cargo stations, doing VTOL on the pads whilst space-trucks are caught by the nearby G.O.O.S.E catapult launcher/catcher. I don't think we need to see the full animation of how cargo is loaded onto the birds, but it might be cool to have a cargo "drop" animation that looks like it's laying an egg! 

### Game Over

:heart: this is spectacular! It's almost ready to use as-is for the Game Over screen! It will really look great with a "glitch"-type flickering animation. It makes me want to drop everything and write the accompanying music :smile:

### Pilot 2

Seeing the poses with and without space-suits is helpful. I like these designs for so many different things! Good idea about using some for NPC's -- they could be dock-workers working to load and unload space-containers into local distribution birds, working alongside the cat characters who are "herding" traffic. That's kind of funny, since cats are notoriously impossible when it comes to that kind of thing. 

Do you see this as being the Player character model? As you probably know,  for the phase 2 FPS-view, that would mean we'd need to be able to essentially just see the hands and parts of the torso and legs.

**Comment by IGIUK** [comment]
![WhatsApp Image 2021-02-10 at 02 50 15 (1)](https://user-images.githubusercontent.com/75528351/107445959-894bcf00-6b4e-11eb-839a-0db7510e06bf.jpeg)
![Uploading WhatsApp Image 2021-02-10 at 02.50.15.jpeg…]()

**Reply by jelster** [reply]
The glowing light-antennae things are a great touch that complements the flags! I like the details like having a steaming styrofoam coffee/tea mug. I definitely can see this design moving forward and being used

**Comment by IGIUK** [comment]
![WhatsApp Image 2021-02-10 at 02 50 16](https://user-images.githubusercontent.com/75528351/107445970-8f41b000-6b4e-11eb-9697-48aa9fbcfbc7.jpeg)

**Reply by jelster** [reply]
These comets could easily find a place in any phase of the game. I see these being in the background of phase 1, potentially interacting with the player in phase 2, and being a cool animation as part of the scoring display in phase 3. 

I think the meshes for this might be procedurally generated, similar to how the asteroids are. A particle system similar to the one described in #24 can be used to create a sparkling tail with the core of the comet being the emitter

**Comment by IGIUK** [comment]
![WhatsApp Image 2021-02-10 at 02 50 15](https://user-images.githubusercontent.com/75528351/107446007-a2548000-6b4e-11eb-8d08-1a17773d18dd.jpeg)

**Reply by jelster** [reply]
Very interesting idea to make different types of amoebas that can help or harm - I really like the extra dimensionality that can add!

So we could have the different types of amoebas, and when the player collides with them they will act as either helpful or harmful pickups! 

The harmful and space-rock actors could cause the player's truck to suffer damage or debuff it by e.g., slowing the truck down. This is a great refinement - nice work on incorporating feedback from the first round! (this goes in general for all of these, in case it wasn't clear!)

**Comment by IGIUK** [comment]
![e9265f88-b877-4059-93b0-a8de1b4b5de2](https://user-images.githubusercontent.com/75528351/107862469-1dbf7580-6e5e-11eb-8f65-7b79538f946b.jpg)
![ec8052fb-78ad-477b-8984-1ade4d3d9171](https://user-images.githubusercontent.com/75528351/107862474-20ba6600-6e5e-11eb-87e0-1dac0938891d.jpg)
![ce3a2e21-6556-4a77-8979-09ce99c82dd4](https://user-images.githubusercontent.com/75528351/107862475-2152fc80-6e5e-11eb-9714-a0878444a497.jpg)

**Reply by jelster** [reply]
Hi - sorry I didn't get back sooner on this. 

These latest sketches are starting to gain more detail. I really appreciate the effort and time you're putting into these! I've enjoyed exploring these ideas stylistically, and it's helped me gain clarity. 

I think that we've drifted a bit too far away from faux-techno- realism and too organic, fantasy in theme. The game's central theme centers around big semi-trucks hauling trailers, and it's tough to picture how that fits in with the current direction of your concept work. 

I'm also concerned how much adaptation or bespoke DCC work would be needed to make the world you've drawn come to life and do good service to your art 🙂

Take a listen to the [main theme song posted here](https://github.com/jelster/space-truckers/discussions/36) if you haven't already - I think it will help convey what I'm getting at, and please ask questions! I'm learning a lot and I won't always be clear or helpful in communicating, but I'm always ready to listen!
